server/odc-service/src/main/java/com/oceanbase/odc/service/task/util/JobUtils.java

```diff
@@ -52,11 +53,25 @@ public static String toJson(Object obj) {
 
     public static int getPort() {
         String port = SystemUtils.getEnvOrProperty(JobEnvConstants.ODC_SERVER_PORT);
-        return port == null ? 8989 : Integer.parseInt(port);
+        return port != null ? Integer.parseInt(port) : 8989;
     }
 
     public static String getLogPath() {
         String logPath = SystemUtils.getEnvOrProperty(JobEnvConstants.LOG_DIRECTORY);
         return logPath == null ? "./log" : logPath;
     }
+
+    public static ConnectionConfig getMetaDBConnectionConfig() {
```

Comment:
Why executor need MetaDB config?

Reply:
DLM need metadb to save data when user restart job